### ISSUE TITLE
RDS popup: improve mobile support and accessibility

### ISSUE THREAD

**Issue by TomaszGasior** [TomaszGasior/RadioLista-v3#249]
The user is able to specify additional information about RDS which is not visible in radio list's table. Only first frame of RDS PS is shown in radio list's table. Additional information is available in small popup shown when first RDS PS frame is hovered or focused.

What needs to be fixed:

- The RDS popup is not available for screen readers. Blind user is unable to read its contents.
- The RDS popup position is hardcoded to the left side of RDS frame inside table cell. When RDS column in the radio list is set to the left side, RDS popup is inconvenient to use.
- The RDS popup does not handle mobile devices at all. It doesn't fit on mobile screen and it is unable to position itself in any useful way.

Replaces https://github.com/TomaszGasior/RadioLista-v3/issues/104
Replaces part of https://github.com/TomaszGasior/RadioLista-v3/issues/209

Extra info from the user https://github.com/TomaszGasior/RadioLista-v3/discussions/247

<img width="1185" height="741" alt="Image" src="https://github.com/user-attachments/assets/76757f79-1e6d-4e26-9f7b-a5c3084a5255" />

<img width="720" height="1600" alt="Image" src="https://github.com/user-attachments/assets/e2ee42ca-c195-4104-830f-efb70336d03b" />
